e2e/tests/localAuthoritySubmitsApplication_test.js

```diff
@@ -42,7 +42,7 @@ Scenario('local authority sees task list', async (I, caseViewPage) => {
 Scenario('local authority changes case name @create-case-with-mandatory-sections-only', async (I, caseViewPage, changeCaseNameEventPage) => {
   await caseViewPage.goToNewActions(config.applicationActions.changeCaseName);
   changeCaseNameEventPage.changeCaseName('New case name');
-  await I.completeEvent('Save and continue');
+  await I.seeCheckAnswers();
```

Comment:
Remember to update to call the right method - seeCheckYourAnswersAndCompleteEvent

Reply:
oh yeees. haha!!